.github/workflows/csharp-test.yml

```diff
@@ -18,13 +18,16 @@ jobs:
       - name: Checkout repository
         id: checkout-stdb
         uses: actions/checkout@v4
+        
+        # Copy global.json to the current directory so that the dotnet things use it
+      - run: cp modules/global.json global.json
```

Comment:
Do we really want to do this?
There are a lot of mostly identical `global.json`s in the tree, we should maybe just use one. `dotnet` looks in parent directories for a `global.json`.

See #3297

Reply:
I do like that approach better. I think if we rename into that PR, the only thing that this PR needs to do is to free up the disk space.